### PR TITLE
Issue1724 autocomplete result list

### LINKED ISSUE #1724: Autocomplete result list improvements
* Autocomplete dropdown list needs to be customizable in length, with a scroll bar
* The background should not be white, as it was considered confusing
* The right side border should be in line with the search button

### PR DESCRIPTION
**Pull request by joelit**
## Reasons for creating this PR

Reacting to user feedback

## Link to relevant issue(s), if any

- Closes #1724 

## Description of the changes in this PR
* Search result list background changed to darker color
* Search result list width according to the search field width
* Add a scroll bar if the search result list or the languagage selection list is longer than 50% of view window height

## Known problems or uncertainties in this PR

* The layout changes need to be approved

## Checklist

- [x] phpUnit tests pass locally with my changes
- [x] I have added tests that show that the new code works, or tests are not relevant for this PR (e.g. only HTML/CSS changes)
- [x] The PR doesn't reduce accessibility of the front-end code (e.g. tab focus, scaling to different resolutions, use of `.sr-only` class, color contrast)
- [x] The PR doesn't introduce unintended code changes (e.g. empty lines or useless reindentation)

### CODE REVIEW

**Review by osma — COMMENTED** [review]
The changes so far look good, although I'm not sure if the darker background is such a great idea. It looks a bit confusing IMHO.

If I may make an additional request, I think the left margin/padding of search results should be the same as in the search field. Currently they don't align very well, as shown here with a red line:

image

File: resource/css/skosmos.css [hunk 1]
```diff
@@ -204,6 +205,7 @@ body {
   }
 
   #search-wrapper {
+    display: flex;
```
Comment (UnniKohonen) [inline resource/css/skosmos.css:208]: This line seems to be overwritten by something else. If flex is needed for the element, `!important` should be added.

**Review by UnniKohonen — CHANGES_REQUESTED** [review]
There is one line in the CSS that seems to be redundant, otherwise looks good to me!

File: resource/css/skosmos.css (hunk 1 above)
Comment (joelit) [reply resource/css/skosmos.css:208]: Damn, that was the only thing that remained when I was trying out a different layout solution. It was needed then, but it seems it is not needed in the current layout.

**Pull request merged**